backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -898,7 +898,7 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
       withSQLConf("spark.sql.autoBroadcastJoinThreshold" -> "1MB") {
         runQueryAndCompare(
           """
-            |select * from t1 cross join t2;
+            |select * from t1 left outer join t2;
             |""".stripMargin
         ) {
           checkOperatorMatch[BroadcastNestedLoopJoinExecTransformer]
```

Comment:
Would you like to explain a bit about the reason changing the test case? Thanks.

Reply:
Added expression matching to enhance the test case, previously it was not covered.